src/client/containers/SelectionPage/SelectionPage.component.js

```diff
@@ -1,5 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './Selectionpage.css';
+import PropTypes from 'prop-types';
+import Radio from '../../components/Radio/ButtonRadio';
+import HyfBigLogo from '../../components/HyfBigLogo/HyfBigLogo';
+import { Button } from '../../components/Button/Button';
 
-export default function SelectionPage() {
-  return <h1>This is a SelectionPage</h1>;
-}
+const SelectOption = (props) => {
+  const { onClick } = props;
+  const submit = () => {
+    onClick();
+  };
+  const [Title, setTitle] = useState('web dev rooms');
```

Comment:
I know you used this for testing, but you should not commit such code.

Reply:
![title](https://user-images.githubusercontent.com/36760389/102259594-9615c300-3f0f-11eb-94a3-f669d46119f0.PNG)
actually this is what i wanted to do when you choose student title will change to 'Web dev rooms' as it is on this page and when you choose mentor it will change as it is on figma

Reply:
Wow! I didn't even notice that, good eye. But still, I don't think we need it.

Reply:
Also, variables should be named with lowercase so `title`.